ledger/catchpointtracker.go

```diff
@@ -62,6 +66,11 @@ const (
 	trieAccumulatedChangesFlush = 256
 	// CatchpointDirName represents the directory name in which all the catchpoints files are stored
 	CatchpointDirName = "catchpoints"
+
+	// CatchpointFileVersionV5 is the catchpoint file version that was used when the database schema was V0-V5.
+	CatchpointFileVersionV5 = uint64(0200)
+	// CatchpointFileVersionV6 is the catchpoint file version that is matching database schema V6
+	CatchpointFileVersionV6 = uint64(0201)
```

Comment:
We should make it clear that v6  is the same as EnableAccountDataResourceSeparation and appeared on accountDataResourceSeparationRound